docs/testing_with_web3j_unit.md

```diff
@@ -0,0 +1,127 @@
+Web3j Unit
+===========
+
+A web3j-unit is an extension to [JUnit 5](https://junit.org/junit5/docs/current/user-guide/) which enables you to test solidity contracts like any other Java code.
+Rather than having to spin up a node, or use Ganache for integration testing, Web3j-unit runs an EVM in process.
+Web3j-unit also provides you with the flexibility of testing against specific Geth, Besu or Parity nodes.
+A docker-compose network can also be spun up and used for testing contracts using web3j-unit.
```

Comment:
I think maybe these three lines should be put together a bit differently. Maybe something like:

Web3j-unit allows you to run with both an embedded and dockerized Ethereum nodes, with out of the box support for Geth, Besu, and Parity. A docker-compose network can also be configured easily for more complex setups.